AndrewTehan/hometask-4/my_hash.rb

```diff
@@ -15,7 +15,7 @@ def initialize
   def []=(key, value)
     resize! if full_dictionary?
 
-    slot = slot(2)
+    slot = slot(key)
```

Comment:
could you pls remove changes that are not connected to the task and move them in a separate PR?